tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -431,6 +431,105 @@ def test_get_metric_extraction_config_with_apdex(default_project):
         }
 
 
+@django_db_all
+@pytest.mark.parametrize("quality", ["good", "meh", "poor", "any"])
+@pytest.mark.parametrize("measurement", ["measurements.lcp"])
```

Comment:
Do we want to test out the other measurements? I think we're fine.
https://github.com/getsentry/sentry/blob/5042b1ab880ac138d33f69811533abcc497b8662/src/sentry/search/events/constants.py#L61-L82

Reply:
Yeah I think it's fine, I might add one more measurement just to make sure it's not special cased, but not going to do all of them